tests/sanity.js

```diff
@@ -42,3 +42,11 @@ assert(new IntlPolyfill.DateTimeFormat('en', {
     month:'long',
     year: 'numeric'
 }).format(new Date('2016/01/16')), 'January 2016', 'month should be long');
+
+// Issue #192
+assert((function () {
```

Comment:
@headcr4sh align with the other tests, and I can merge this in preparation for that fix.